content/en/news/security/istio-security-2022-007/index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: ISTIO-SECURITY-2022-007
+subtitle: Security Bulletin
+description: Denial of service attack due to Golang Regex Library.
+cves: [CVE-2022-39278]
+cvss: "7.5"
+vector: "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H"
+releases: ["All releases prior to 1.13", "1.13.0 to 1.13.8", "1.14.0 to 1.14.4", "1.15.0 to 1.15.1"]
```

Comment:
Given we have a CVE, should we rewrite the release announcements to include the CVE similar to the release notes in the past? Today they talk about the Go vulnerabilities but not Istio's vulnerabilities.

We also need to update the table at `docs/releases/supported-releases/` to match this note.

Reply:
yeah we can add the CVE details in our release notes, will update the `docs/releases` part.

Reply:
Basically it's enough to add the CVE details in the release notes right ?

Reply:
> Basically it's enough to add the CVE details in the release notes right ?

Yes. If there was an Istio CVE for a release and we have a security news item, it's typically called out like this:
<img width="775" alt="Screenshot 2022-10-14 at 12 46 01 PM" src="https://user-images.githubusercontent.com/10537847/195909212-f5c148ef-4897-45a4-

Someone reading the release notes would see that there is an Istio CVE and likely update.

The current release notes imply there might have been a CVE in Go, but we might not be affected:
<img width="736" alt="Screenshot 2022-10-14 at 12 47 43 PM" src="https://user-images.githubusercontent.com/10537847/195909424-9d856ea0-d93d-4f0c-accb-f25ecccbeae3.png">
9621-4ac91ceb3d80.png">

Reply:
@lei-tang @Monkeyanator could you please update the release notes based on this comment. It's good if we add the CVE details in our release notes.